tests/algorithms/test_adversarial.py

```diff
@@ -293,7 +303,7 @@ def test_logits_gen_is_high_log_policy_act_prob(
         trans.next_obs,
         trans.dones,
     )
-    log_act_prob_non_none = th.as_tensor(np.random.rand(n_timesteps))
+    log_act_prob_non_none = th.as_tensor(np.random.rand(n_timesteps)).to(obs.device)
```

Comment:
Perhaps the log probs should be negative just for the sake of realism? e.g.

```suggestion
    log_act_prob_non_none = th.as_tensor(np.log(0.1 + 0.9 * np.random.rand(n_timesteps))).to(obs.device)
```

Reply:
(not related to this PR, just seems a bit odd passing in "log probs"  that are not actually log probs)